modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupRelLocalServiceImpl.java

```diff
@@ -158,36 +152,36 @@ public AccountGroupRel fetchAccountGroupRel(
 			classPK);
 	}
 
+	/**
+	 * @deprecated As of Cavanaugh (7.4.x), replaced by {@link #getAccountGroupRelsByAccountEntryId(String, long)}
+	 */
+	@Deprecated
 	@Override
-	public List<AccountGroupRel>
-		getAccountGroupRelsByAccountEntryId(long accountEntryId) {
+	public List<AccountGroupRel> getAccountGroupRelsByAccountEntryId(
+		long accountEntryId) {
 
-		return accountGroupRelPersistence.findByAccountEntryId(
-			accountEntryId);
+		return getAccountGroupRelsByAccountEntryId(
+			AccountEntry.class.getName(), accountEntryId);
 	}
 
 	@Override
-	public List<AccountGroupRel>
-	getAccountGroupRelsByAccountEntryId(String className, long classPK) {
+	public List<AccountGroupRel> getAccountGroupRelsByAccountEntryId(
```

Comment:
This should be called `getAccountGroupRels` because it takes in the `className` and `classPK`.